src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import ArticleCard from "./ArticleCard";
+
+function ArticlesList() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(`https://news-api-ibvn.onrender.com/api/articles`)
+      .then((response) => {
+        if (response.data.articles) {
+          console.log("this is response from data", response.data.articles);
+          setArticles(response.data.articles);
+        } else {
+          setArticles([]);
+        }
```

Comment:
There's no need for the conditional logic here. The endpoint will respond with a list of articles unless your backend is not working.